visualizer/semanticsEditor.js

```diff
@@ -360,6 +366,177 @@
   }
   ohmEditor.parseTree.addListener('cmdclick:traceElement', toggleSemanticsEditor);
 
+  // Retrieve arguments from the menu entry's submenu
+  function retrieveArgumentsFromSubList(entry) {
+    var argumentWrappers = entry.querySelectorAll('ul li');
+    if (!argumentWrappers) {
+      return undefined;
```

Comment:
Ah, dang. Maybe we should think about turning that check off -- I'm not sure how useful it is. Don't worry about it for now though, just leave this code as-is.